jboss-modules/build.xml

```diff
@@ -146,6 +146,13 @@
         <module-def name="org.jboss.resteasy.resteasy-jackson2-provider">
             <maven-resource group="org.jboss.resteasy" artifact="resteasy-jackson2-provider"/>
         </module-def>
+    	
+        <module-def name="org.jboss.resteasy.resteasy-rxjava">
+            <maven-resource group="org.jboss.resteasy" artifact="resteasy-rxjava"/>
+        	<maven-resource group="io.reactivex" artifact="rxjava"/>
+        	<maven-resource group="io.reactivex" artifact="rxjava-reactive-streams"/>
+        	<maven-resource group="org.reactivestreams" artifact="reactive-streams"/>
+        </module-def>

```

Comment:
We already have modules containing the rxjava and reactive-streams libraries on the container, so I believe we should be referencing them instead of bundling the libraries in our resteasy-rxjava module.